src/components/project.js

```diff
@@ -0,0 +1,198 @@
+import { Header, HeaderLineBelow } from "../styles/Header"
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import Body from "../styles/Body.js"
+import { SB_ORANGE, SB_NAVY_RGBA } from "@colors"
+
+const CardBG = styled.div`
+  margin-bottom: 20px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  border-radius: 20px;
+  transition: 0.5s;
+  width: 300px;
+  height: 300px;
+  text-align: center;
+  &:hover {
+    transform: translateY(-5px);
+  }
+`
+
+const Card = styled.div`
+  margin-top: -20px;
+  width: 300px;
+  height: 300px;
+  border-radius: 20px;
+  background-color: ${SB_NAVY_RGBA(0.8)};
+`
+
+const CardHeader = styled(Header)`
+  padding-top: 45px;
+  font-size: 1.5em;
+  font-weight: 300;
+  color: ${SB_ORANGE};
+  height: 100px;
+  margin-left: 20px;
+  margin-right: 20px;
+`
+const TagsContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 75%;
+  margin: auto;
+  align-items: center;
+  justify-content: center;
+  height: 50px;
+`
+
+const Tag = styled.div`
+  font-size: 0.9em;
+  display: inline-block;
+  border-radius: 4px;
+  background-color: rgba(0, 0, 0, 0.35);
+  padding-right: 7px;
```

Comment:
You'll save a lot of LOC in this file by using shortcuts for padding and margin. When you use the css attribute `padding: a b c d;` it's a shortcut for 
```css
padding-top: a;
padding-right: b;
padding-bottom: c;
padding-left: d;
```
and same for `margin`. This comment applies throughout the file.